src/room/InCallView.tsx

```diff
@@ -98,7 +98,7 @@ export function ActiveCall(props: ActiveCallProps) {
   const livekitRoom = useLiveKit(props.userChoices, {
     sfuUrl: Config.get().livekit!.server_url,
     jwtUrl: `${Config.get().livekit!.jwt_service_url}/token`,
-    roomName: props.matrixInfo.roomName,
+    roomName: `{props.matrixInfo.roomName}({props.matrixInfo.roomId})`,
```

Comment:
Note that without `$` before the braces, this is actually the literal string `"{props.matrixInfo.roomName}({props.matrixInfo.roomId})"`

Reply:
Oh, I was totally blind when reviewing this...

Reply:
My bad, thanks, fixed!